app/views/foreman/job_templates/install_errata.erb

```diff
@@ -15,5 +15,10 @@ foreign_input_sets:
   exclude: action,package
 %>
 
-<% advisories = input(:errata).split(',').map { |e| "--advisory=#{e}" }.join(' ') %>
-<%= render_template('Package Action - SSH Default', :action => 'update-minimal', :package => advisories) %>
+<% if @host.operatingsystem.family == 'Suse' -%>
+    <% advisories = input(:errata).split(',').join(' ') %>
+    <%= render_template('Package Action - SSH Default', :action => 'install -n -t patch', :package => advisories) %>
+<% else %>
+    <% advisories = input(:errata).split(',').map { |e| "--advisory=#{e}" }.join(' ') %>
+    <%= render_template('Package Action - SSH Default', :action => 'update-minimal', :package => advisories) %>
+<% end %>
```

Comment:
Is the goal here to execute 'zypper install -n -t patch errata1 errata2'  ?  If so, wouldn't the package action template in remote_execution need updating for suse?  https://github.com/theforeman/foreman_remote_execution/blob/master/app/views/templates/ssh/package_action.erb

it looks like it only supports redhat and debian currently.

Reply:
the package_action in the community-template does have the suse support. Thought the community template is merged to katello. remote_execution automatically. It's a confusing to have multiple sources for the same template.

Reply:
ahhhh thank you !  Will go ahead and merge this and whenever that gets pulled in it'll start working.